docs/tutorial/application-distribution.md

```diff
@@ -36,47 +50,43 @@ electron/resources/app
 └── index.html
 ```
 
-Then execute `Electron.app` (or `electron` on Linux, `electron.exe` on Windows),
+Then execute `Electron.app` (`electron` on Linux, `electron.exe` on Windows),
 and Electron will start as your app. The `electron` directory will then be
-your distribution to deliver to final users.
+your distribution to deliver to users.
 
-## Packaging Your App into a File
+### With asar archive
```

Comment:
Putting "asar" in the heading is kind of weird because no one knows what it is until they read the section. It's probably better to describe what it does instead

```suggestion
### With an app source code archive
```